src/client/getOrCreateEditor.js

```diff
@@ -68,8 +68,7 @@ export const getOrCreateEditor = ({
   }
 
   let editor = editorCache.get(fileId);
-  if (!editor) {
-    // Cache miss --> mint a new editor.
+  if (!editor || editor) {
```

Comment:
This doesn't look right. We only want to mint a new editor if there is no existing editor. This is the same as `if(true)`.
```suggestion
  if (!editor) {
```

Maybe let's change it back, and figure out how to swap out the theme plugin on the already created editor. I think there should be a way.

I researched this a bit and it looks like "Compartments" is what we need. See https://codemirror.net/examples/config/#dynamic-configuration